common/main/root.xml

```diff
@@ -6172,6 +6172,50 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 				<unitPattern count="other">{0} night</unitPattern>
 				<perUnitPattern>{0}/night</perUnitPattern>
 			</unit>
+<unit type='concentr-katal'><displayName>kat</displayName><unitPattern count='other'>{0} kat</unitPattern></unit>
```

Comment:
Not indented?

Reply:
I have that in the notes. As a final step before submission, I'll use CLDRModify to clean up the file. However, that will also distribute these all over the place, so during development (over a couple of different PRs and tickets), it is easier to keep them in one place.